plugin/storage/es/esRollover.py

```diff
@@ -80,7 +99,12 @@ def perform_action(action, client, write_alias, read_alias, index_to_rollover, t
 def create_index_template(template, template_name):
     print('Creating index template {}'.format(template_name))
     headers = {'Content-Type': 'application/json'}
-    r = requests.put(sys.argv[2] + '/_template/' + template_name, headers=headers, data=template)
+    s = requests.Session()
+    if str2bool(os.getenv("ES_TLS", 'false')):
```

Comment:
Does the username/password need to also be supported here?

Reply:
probably yes, they were not supported in this script at all.

I will add them to keep the parity with cleaner script.